cli/cli.go

```diff
@@ -1,456 +1,822 @@
+	// Default value is set in the template
+	o.genData.MevImage, err = p.Input("Mev-Boost image", "flashbots/mev-boost:latest", false)
+	return
+}
+
+func inputMevBoostEndpoint(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.genData.MevBoostEndpoint, err = p.Input("Mev-Boost endpoint", "", false)
+	return
+}
+
+func inputRelayURL(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// TODO add default relay URL value, it is not present in the generate command
+	o.genData.RelayURL, err = p.Input("Relay URL", "", false)
+	return
+}
+
+func inputGraffiti(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.genData.Graffiti, err = p.Input("Graffiti", "", false)
+	return
+}
+
+func inputCheckpointSyncURL(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// Default value is set in the template
+	o.genData.CheckpointSyncUrl, err = p.Input("Checkpoint sync URL", "", false)
+	return
+}
+
+func inputFeeRecipient(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// TODO: Is necessary to add a default value?
+	o.genData.FeeRecipient, err = p.EthAddress("Please enter the Fee Recipient address. You can leave it blank and press enter (not recommended)", "")
+	return
+}
+
+func inputValidatorGracePeriod(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	epochs, err := p.InputInt64("Validator grace period (epochs)", 1)
+	if err != nil {
+		return err
+	}
+	o.genData.VLStartGracePeriod = uint(epochs * int64(configs.NetworkEpochTime(o.genData.Network)))
 	return nil
 }
+
+func inputGenerationPath(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// TODO: add default
+	o.generationPath, err = p.Input("Generation path", configs.DefaultAbsSedgeDataPath, false)
+	if o.generationPath == "" {
+		o.generationPath = configs.DefaultAbsSedgeDataPath
+	}
+	return
+}
+
+func inputJWTPath(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.genData.JWTSecretPath, err = p.InputFilePath("JWT path", "", true)
+	return
+}
+
+func inputKeystoreMnemonicPath(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.keystoreMnemonic, err = p.Input("Mnemonic path", "", true)
+	return
+}
+
+func inputKeystorePassphrasePath(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.keystorePassphrasePath, err = p.Input("Passphrase path", "", true)
+	return
+}
+
+func inputKeystorePassphrase(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.keystorePassphrase, err = p.InputSecret("Passphrase")
+	return
+}
+
+func inputWithdrawalAddress(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.withdrawalAddress, err = p.Input("Withdrawal address", "", false)
+	return
+}
+
+func inputNumberOfValidators(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.numberOfValidators, err = p.InputInt64("Number of validators", 1)
+	return
+}
+
+func inputNumberOfExistingValidators(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.existingValidators, err = p.InputInt64("Existing validators", 0)
```

Comment:
Explain how this number affects the index of the keys to be created

Reply:
Done https://github.com/NethermindEth/sedge/pull/228/commits/4901e69b428639c92c0c9b2ea929ea7f50acd7d9